src/inspector_agent.cc

```diff
@@ -367,14 +317,18 @@ class NodeInspectorClient : public v8_inspector::V8InspectorClient {
   std::unique_ptr<ChannelImpl> channel_;
 };
 
-AgentImpl::AgentImpl(Environment* env) : parent_env_(env),
-                                         inspector_(nullptr),
-                                         platform_(nullptr),
-                                         inspector_console_(false) {}
+Agent::Agent(Environment* env) : parent_env_(env),
+                                 inspector_(nullptr),
+                                 platform_(nullptr),
+                                 inspector_console_(false) {}
+
+// Header has unique_ptr to some incomplete types - this definition tells
+// the compiler to figure out destruction here, were those types are complete
+Agent::~Agent() {
```

Comment:
I think I get it now, perhaps the comment is a little vague.  The "incomplete types" refers to Agent's unique_ptr data members, not to a `std::unique_ptr<Agent>` instance elsewhere, correct?

Vague or not, it should at least be punctuated. :-)